krate/src/main/kotlin/hu/autsoft/krate/Functions.kt

```diff
@@ -100,3 +105,123 @@ public fun Krate.stringPref(key: String, defaultValue: String): ReadWritePropert
 public fun Krate.stringSetPref(key: String, defaultValue: Set<String>): ReadWriteProperty<Krate, Set<String>> {
     return StringSetDelegateWithDefault(key, defaultValue)
 }
+
+/**
+ * Creates a validated optional preference of type [Long] with a given [key]
+ *
+ * @param [key] the key to identify this value in SharedPreferences
+ * @param [isValid] the function to validate values during set
+ *
+ * @return [ReadWriteProperty]
```

Comment:
Same comment as above, I don't think `@param` and `@return` are necessary. I'd go with just a description like this, which refers to the parameters:

```kotlin
/**
 * Creates a validated, optional preference of type Long with the given [key].
 * 
 * If a value being set to this preference returns `false` when checked by [isValid],
 * an exception will be thrown.
 */
```